app/src/main/java/com/android/template/utils/calendar/CalendarUtils.kt

```diff
@@ -0,0 +1,28 @@
+package com.android.template.utils.calendar
+
+import java.util.*
+
+// TODO: maybe somehow merge with DateUtils from domain? Does that make any sense?
```

Comment:
Možda bi stvarno bilo bolje koristiti DateUtils pa u bazu zapisati timestamp po onim UTC pravilima pa da se onda tako parsira, nisam pametan. Koristim Calendar trenutno jer sam s tim prije radio pa znam da nije teško i mogu sa SimpleDateFormat parsirati bilo kakav string (ponekad u bazi imam samo godina-mjesec-dana, ponekad samo sate, a ponekad sve troje, ako koristim DateUtils ima redundancije kod zapisivanja što ne bi trebao biti problem).